gui/resources.xml

```diff
@@ -763,27 +784,35 @@ Vocabulary
       <!-- {{{ inktool points related actions -->
       <child>
         <object class="GtkAction" id="InsertCurrentNode">
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a node back of the currently selected node.</property>
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Insert Inking Node</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a inking node on the back of the currently selected node.</property>
```

Comment:
The button calls it "Insert Point". "Node" is a bit technical.

"[In/on the] back of" isn't comprehensible to British English speakers, and I think it's factually incorrect too if it means "behind"! The node is inserted after the selected node, in timewise/strokewise ordering (= towards the tick mark)